pkg/ingester-rf1/objstore/storage.go

```diff
@@ -31,6 +31,10 @@ func New(
 	storageConfig storage.Config,
 	clientMetrics storage.ClientMetrics,
 ) (*Multi, error) {
+	if storageConfig.GCSConfig.ChunkBufferSize == 0 {
```

Comment:
note: we're already passing by value here so this config is not the same struct as in the top level modules.go

Reply:
storageConfig yes but are we sure this is the case for `GCSConfig` ?